packages/core/cli/deploy.ts

```diff
@@ -66,14 +66,11 @@ const deploy = async (args: any) => {
     const _maxReputationBudget = MAX_REPUTATION_BUDGET
```

Comment:
Why you don't use MAX_REPUTATION_BUDGET directly? Because we don't reassign `_maxReputationBudget` anymore.
Uhm, maybe I wrote this line.
 Should we remove it?

Reply:
yeah there is no need here

Reply:
yeah there is no need here

Reply:
Sure